src/models/hook.js

```diff
@@ -23,6 +23,11 @@ const Hook = DataConnected.extend({
     return utils.deepExtend(this._super(), defaults);
   },
 
+  init(name, values, parent, bind) {
+    this._super(name, values, parent, bind);
+    if (!utils.isEmpty(this._parent._space)) this.setInterModelListeners();
```

Comment:
setInterModelListeners() should not be called in init phase on model tree because other models are still initialising

if you look in [tool.js](https://github.com/vizabi/vizabi/blob/develop/src/base/tool.js#L149), first we need to build the model and only then make connections between its parts

why do you need to do this in init? maybe you could just wait until it will be called across the tree after init?

Reply:
this for case when hook is added after tool model was created. But may be better call label.setInterModelListener after new label is added in marker.setSpace ?

Reply:
Probably better. This influences general hook init behaviour, which we don't want.